design/snapshots.md

```diff
@@ -45,7 +45,7 @@ interface.
 ## Architecture
 
 The _Snapshot Manager_ provides an API for allocating, snapshotting and mounting
-abstract, layer-based filesytems. The model works by building up sets of
```

Comment:
Ha, when [misspell](https://github.com/client9/misspell) told me:
`design/snapshots.md:48:22: "filesytems" is a misspelling of "filesystem"`, I thought it's talking about plural issue :P , fixed.